### PR TITLE
Modified database models 

### PR DESCRIPTION
**Pull request by EmeraldEntities**
In regards to the comments brought up in #15, the database was slightly altered to promote the efficiency of indexing and allow for simple retrieval of images.

Images each have their own UUID which can be used to identify them when retrieving them using the API URI (see #16 ). Also, moving them to the table should improve query times on `Board` and `Post`.

### CODE REVIEW

**Comment by shari09** [comment]
I don't think it improves query times on `Board` and `Post` ~unless I'm wrong~

File: board/tests.py [hunk 1]
```diff
@@ -10,16 +12,60 @@ class BoardModelTests(TestCase):
...
+    p1 = Post(associated_board=b2, name="p1", message="m1")
+    p2 = Post(associated_board=b2, name="p1", message="m1", photo=Image(name="spam", photo=bytearray("eggs", "utf-8")).save())
+    p3 = Post(associated_board=b2, name="p1", message="m1", photo=i3)
+    p1.save()
+    p2.save()
+    p3.save()
+
+    self.assertIs(p3.photo, i3)
+    self.assertIs(i3.post, p3)
+    self.assertIs(p2.photo.name, "spam")
```
Comment (shari09) [inline board/tests.py:50]: Can you write tests that throws exceptions and test those? Maybe I'll make a new issue for that.

File: board/models.py [hunk 2]
```diff
@@ -2,10 +2,47 @@
...
+
+    Class Attributes
+        name -> `CharField`: A charfield with max length 100 with the name of the image.
+        photo -> `BinaryField`: A field with the stored image BLOB.
+        uuid -> `UUIDField`: A unique, non-editable uuid4 UUID for each image, used to locate it.
+    """
+
+    name = models.CharField(max_length=100)
+    photo = models.BinaryField()
+    uuid = models.UUIDField(default=uuid.uuid4, unique=True, editable=False)
```
Comment (shari09) [inline board/models.py:26]: Can you also add a `created_at` field for the images (one day it'll probably be useful).

**Review by shari09 — CHANGES_REQUESTED** [review]
Just one comment

File: board/models.py (hunk 2 above)
Comment (EmeraldEntities) [reply board/models.py:26]: :ok_hand:

File: board/tests.py (hunk 1 above)
Comment (EmeraldEntities) [reply board/tests.py:50]: 👌

**Review by shari09 — APPROVED** [review]
niceeeee 😄

**Pull request merged**